index.js

```diff
@@ -8,6 +8,7 @@ var defaults = {
   trustAzureHeader: false,
   port: 443,
   hostname: null,
+  skipPort: false,
```

Comment:
Not sure we have to change this. It's breaking changes. I prefer without the port, but breaking changes, are you ok to assume it ?